src/components/Document/write/EmployeeInfoSection.tsx

```
@@ -24,68 +22,65 @@ const EmployeeInfoSection = ({
-                }
-              >
-                {!['address', 'signature_base64'].includes(key) && (
-                  <div className="w-full self-stretch drop-shadow-[0_1px_2px_rgba(107,110,116,0.04)] rounded-xl flex items-center justify-start py-2.5 pr-3.5 pl-4 body-2">
-                    <div className="w-full flex-1 relative">
-                      {propertyToString(String(value))}
-                    </div>
-                  </div>
-                )}
-              </InputLayout>
+            <div className="w-full flex flex-col gap-1">
+              <p className="button-2 text-text-alternative">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)